### PR TITLE
Show empty readme message

### LINKED ISSUE #408: readme for dex chart always loading
The dex chart doesn't have a README.md (https://github.com/helm/charts/tree/master/stable/dex) but it is shown as constantly Loading in the UI:

screen shot 2018-07-23 at 5 55 01 pm

The README.md should have a 404 response

### PR DESCRIPTION
**Pull request by andresmgot**
Fixes #408

### CODE REVIEW

File: dashboard/src/components/ChartView/ChartReadme.test.tsx [hunk 1]
```diff
@@ -4,11 +4,11 @@ import * as ReactMarkdown from "react-markdown";
 
 import ChartReadme from "./ChartReadme";
 
-it("renders a loading message if the readme is not present", () => {
+it("renders a README not found message if the readme is not present", () => {
   const wrapper = shallow(
     <ChartReadme getChartReadme={jest.fn()} hasError={false} version="1.2.3" />,
   );
-  expect(wrapper.text()).toBe("Loading");
```
Comment (migmartri) [inline dashboard/src/components/ChartView/ChartReadme.test.tsx]: Do we have any other test that targets the loading part of the component?

Comment (andresmgot) [reply dashboard/src/components/ChartView/ChartReadme.test.tsx]: I did follow a wrong approach for fixing this. The idea was to throw an error when the README is empty not remove the `Loading` message.

File: Makefile [hunk 2]
```diff
@@ -38,6 +38,10 @@ kubeapps/tiller-proxy:
 	CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -o ./cmd/tiller-proxy/proxy-static ./cmd/tiller-proxy
 	docker build -t kubeapps/tiller-proxy:$(VERSION) -f cmd/tiller-proxy/Dockerfile cmd/tiller-proxy
 
+kubeapps/chartsvc:
```
Comment (andresmgot) [inline Makefile]: This change and the Dockerfile below are unrelated to the bug fix. It's just for building the image faster (2m vs 8s in my local machine)

**Comment by andresmgot** [comment]
@migmartri please take a look again, the approach is now different

File: Makefile (hunk 2 above)
Comment (migmartri) [reply Makefile]: I do not think that this change should be done here and it should probably be part of a bigger discussion. Maybe as part of https://github.com/kubeapps/kubeapps/issues/461 or could you create an issue explaining what problem you are having?

Please do not add unrelated code to a patch still less if it has already been accepted.

Thanks

**Comment by migmartri** [comment]
I prefer your current approach of returning 404 if the readme is not there. Much better. Thanks for the change.

File: Makefile (hunk 2 above)
Comment (andresmgot) [reply Makefile]: had to try :P I'll remove this and open a different issue for that

**Pull request merged**